test/pageobjects/page.ts

```diff
@@ -0,0 +1,7 @@
+import { browser } from "@wdio/globals";
+
+export default class Page {
+  public open() {
+    return browser.url(`https://dev.rentzila.com.ua/`);
```

Comment:
@Anastas11aZakharova I guess, it is better to use the 'base Url' parameter in the configuration. At least, it could be easily changed by the command, just in case.

Reply:
Done in the branch with the next set of tests where baseUrl is used along with the dotEnv to hide the credentials